src/common/utils.ts

```diff
@@ -837,6 +837,18 @@ export const addStatusFields = <T>(
   };
 };
 
+export const addUpdatedJobParamsField = <T>(
+  obj: T,
+  jobParams: Record<string, unknown> | undefined,
+): T & {
+  jobParams: Record<string, unknown> | undefined;
+} => {
+  return {
+    ...obj,
+    jobParams: jobParams,
+  };
+};
+
```

Comment:
I wonder if this function should be in the job controller or service as it is specific for the jobs?